pkgs/development/python-modules/marisa-python/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonPackage, fetchgit, marisa, swig }:
+
+buildPythonPackage rec {
+  pname = "marisa-python";
```

Comment:
Is there another package repo that calls this `marisa-python` or why did you choose that name?

Reply:
At first I thought that it's a good idea since there is a project named `marisa-tie` which is a handwritten(?) python bindings to C++ `marisa` library. And this one is the bindings generated by swig.

Now I think that you are right and it's better to rename it to `marisa` especially since the same package called `python2-marisa` in Arch Linux repos and we have `pythonPackages` namespace.